tests/suites/cloud/tests/os-config/index.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
+	title: 'os-config tests',
+	tests: [
+		{
+			title: 'os-config service on boot',
+			run: async function(test) {
+				let result = '';
+				await this.utils.waitUntil(async () => {
+					result = await this.cloud.executeCommandInHostOS(
+							`systemctl is-active os-config.service`,
```

Comment:
For future reference this logic is available as a helper here:
https://github.com/balena-os/meta-balena/blob/5e9ec7690114640a5a7c3f6c94bb7e3ed37977ab/tests/suites/os/suite.js#L92-L110

Reply:
I'll change this on a subsequent commit @klutchell thanks